src/routes/Private.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+  Nested Routes: https://reactrouter.com/en/main/start/overview
+*/
+
+import { lazy } from 'react';
+
+const ListUser = lazy(() => import('../pages/user/ListUser'));
+const AddUser = lazy(() => import('../pages/user/AddUser'));
+const EditUser = lazy(() => import('../pages/user/EditUser'));
+const EditUserRTKQuery = lazy(() => import('../pages/user/EditUserRTKQuery'));
+
+export default {
```

Comment:
```suggestion
export const privateRoutes = [
  {
    path: '/user',
    component: ListUser,
  },
  {
    path: '/user/add-new',
    component: AddUser,
  },
  {
    path: "/users/:userId/edit",
    component: EditUser,
  }
]
```

export an array like will be easy to handle and cleaner

The fileName should be private-routes.tsx. 

Some rule you can follow

CamelCase: component, views
kebab-case: helpers, service